extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/handler/SacnBridgeHandler.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.dmx.handler;
+
+import static org.eclipse.smarthome.binding.dmx.DmxBindingConstants.*;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Set;
+import java.util.UUID;
+
+import org.eclipse.smarthome.binding.dmx.internal.Util;
+import org.eclipse.smarthome.binding.dmx.internal.dmxoverethernet.DmxOverEthernetHandler;
+import org.eclipse.smarthome.binding.dmx.internal.dmxoverethernet.IpNode;
+import org.eclipse.smarthome.binding.dmx.internal.dmxoverethernet.SacnNode;
+import org.eclipse.smarthome.binding.dmx.internal.dmxoverethernet.SacnPacket;
+import org.eclipse.smarthome.binding.dmx.internal.multiverse.Universe;
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link SacnBridgeHandler} is responsible for handling the communication
+ * with sACN/E1.31 devices
+ *
+ * @author Jan N. Klug - Initial contribution
+ */
+public class SacnBridgeHandler extends DmxOverEthernetHandler {
+    public final static Set<ThingTypeUID> SUPPORTED_THING_TYPES = Collections.singleton(THING_TYPE_SACN_BRIDGE);
+    public final static int MIN_UNIVERSE_ID = 1;
+    public final static int MAX_UNIVERSE_ID = 63999;
+
+    private final Logger logger = LoggerFactory.getLogger(SacnBridgeHandler.class);
+    private final UUID senderUUID;
+
+    public SacnBridgeHandler(Bridge sacnBridge) {
+        super(sacnBridge);
+        senderUUID = UUID.randomUUID();
+    }
+
+    @Override
+    protected void updateConfiguration() {
+        Configuration configuration = getConfig();
+
+        int universeId = MIN_UNIVERSE_ID;
```

Comment:
This was my first try. However, I did not manage to use different MIN_UNIVERSE_ID and MAX_UNIVERSE_ID for different handlers if I parse that in DmxBridgeHandler. If you can point me how to handle that, I'm happy to move it there.